libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -77,125 +77,125 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
     // NOTE: Index numbers 0 and 1 were for the old integer
     // ground temperature and pressure
 
-    // @Param: ABS_PRESS
-    // @DisplayName: Absolute Pressure
+    // @Param: 1_GND_PRESS
+    // @DisplayName: Ground Pressure
     // @Description: calibrated ground pressure in Pascals
     // @Units: Pa
     // @Increment: 1
     // @ReadOnly: True
     // @Volatile: True
     // @User: Advanced
-    AP_GROUPINFO("ABS_PRESS", 2, AP_Baro, sensors[0].ground_pressure, 0),
+    AP_GROUPINFO_FLAGS("1_GND_PRESS", 2, AP_Baro, sensors[0].ground_pressure, 0, AP_PARAM_FLAG_INTERNAL_USE_ONLY),
 
-    // @Param: TEMP
+    // @Param: 1_TEMP
```

Comment:
Don't want to rename it to be BARO1_GND_TEMP while we're modifying it?

Reply:
i've changed it to BARO_GND_TEMP
it is actually not per-baro